src/Rules/Drupal/TrustedCallbackRule.php

```diff
@@ -172,6 +174,13 @@ private function doProcessNode(Node\Expr $node, Scope $scope, string $keyChecked
 
         foreach ($type->getConstantArrays() as $constantArrayType) {
             if (!$constantArrayType->isCallable()->yes()) {
+                if ($constantArrayType->getItemType() instanceof UnionType
+                    && !($constantArrayType->getValuesArray()->getValueTypes()[1] instanceof ConstantStringType)) {
```

Comment:
This makes my eyes bleed, but couldn't find a decent way to determine if the right hand side of the UnionType is not a ConstantStringType, but a StringType, which (in my undestandi9ng of PHPStan magic) makes this a variable.